src/docfx/restore/RestoreMap.cs

```diff
@@ -223,6 +257,19 @@ public static (string path, DependencyGit git) AcquireExclusiveGit(string remote
         public static bool ReleaseGit(DependencyGit git, LockType lockType, bool successed = true)
             => DependencySlotPool<DependencyGit>.ReleaseSlot(git, lockType, successed);
 
+        private static string TryGetRestoredFilePath(string url)
```

Comment:
consider layer `GetRestoredFileContent` on top of `GetRestoredFilePath`

Reply:
it would not save much logic since there are both `content` and `etag` to be prepared